tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -50,12 +50,18 @@ var pc *pbm.Client
 var spareSpace int64 = 200
 
 var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation tests", func() {
-
 	f := framework.NewDefaultFramework("e2e-spbm-policy")
+	if supervisorCluster {
```

Comment:
can you add comments here explaining why we need this

Reply:
In case of supervisorCluster, we already have a namespace which is created in pre-setup, so while running tests we use that pre-created namespace itself. In case of test failure, test framework tries to collect events from the framework created namespace ie. "e2e-spbm-policy" instead of pre-created namespace which results in 0 events. So to overcome this issue, I have made this change